mysql-test/suite/tianmu/t/replace.test

```diff
@@ -0,0 +1,35 @@
+--source include/have_tianmu.inc
+
+#
+# Test REPLACE INTO with TIANMU
+#
+
+--disable_warnings
+drop table if exists test1;
+--enable_warnings
+# test auto_increment and the primary key covers 1 columns
+CREATE TABLE test1 (
+  id INT NOT NULL AUTO_INCREMENT,
+  data VARCHAR(64) DEFAULT NULL,
+  ts TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (id)
+)  engine=tianmu;
+REPLACE INTO test1 VALUES (1, 'Old', '2014-08-20 18:47:00');
```

Comment:
Pls, add a test like:
```
insert into test1 values (1,xxx);
insert into test1 values (2, xxxx);
replace into test1 values(1, xxxx);
delete from test1 where id =1;
replace int test1 (1,xxxx);
```
Thanks. 



```

Reply:
Thank for your advice. I'll make the requested changes!